Dockerfile

```diff
@@ -7,11 +7,17 @@ RUN apk update && apk add py3-psycopg2 musl-dev \
     nginx supervisor git \
     openssl ca-certificates \
     gcc postgresql-dev \
- && pip install --upgrade pip \
- && pip install virtualenv
+ && pip install --upgrade pip
 
 RUN         pip install -r /app/requirements.txt
-ADD         . /app
+ADD         dataservice /app
+ADD         docs /app
```

Comment:
Do you need docs and .git?

Reply:
The status endpoint relies on .git for the commit hash [here](https://github.com/kids-first/kf-api-dataservice/blob/master/dataservice/__init__.py#L159-L179)
The swagger docs require the docs [here](https://github.com/kids-first/kf-api-dataservice/blob/5647b38b8e13c0b6fd839be1cd179ed452ccb5b1/dataservice/api/docs/resources.py#L11-L13)